pkgs/development/tools/prototool/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "prototool";
+  version = "1.8.0";
+
+  src = fetchFromGitHub {
+    owner = "uber";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0m6g5p3jyf0ja5j9rqmw8mhipdqxk7rcxm0plmh65bxk1n22fzjc";
+  };
+
+  modSha256 = "0l4nqb1c1lqqk70l9qwib1azavxlwghi2fv5siwrr5zw4jysz5ml";
+
+  subPackages = [ "cmd/prototool" ];
```

Comment:
```suggestion
```

I'm not sure you need this unless it's failing to build without it.

Reply:
Without this it install other unneeded(?) subpackages
```
🗁  v3apdskhafw22qkv8qyj41x24p4rfw30-prototool-1.8.0
└── 🗁  bin
   ├── 🗋  excited
   ├── 🗋  gen-prototool-bash-completion
   ├── 🗋  gen-prototool-manpages
   ├── 🗋  gen-prototool-zsh-completion
   └── 🗋  prototool
```
even upstream only ships cmd/prototool: 
https://github.com/uber/prototool/blob/454821971ee046298879b9c17d4ab1bdd2a6cee8/Makefile#L109-L110
https://github.com/uber/prototool/blob/454821971ee046298879b9c17d4ab1bdd2a6cee8/Dockerfile#L64-L66